Packages/com.chisel.editor/Chisel/Editor/Editor/Settings/CSGEditorSettings.cs

```diff
@@ -24,6 +24,9 @@ public static class CSGEditorSettings
 		public static float				MoveSnapX		{ get { return Grid.defaultGrid.SpacingX; } set { Grid.defaultGrid.SpacingX = value; } }
 		public static float				MoveSnapY		{ get { return Grid.defaultGrid.SpacingY; } set { Grid.defaultGrid.SpacingY = value; } }
 		public static float				MoveSnapZ		{ get { return Grid.defaultGrid.SpacingZ; } set { Grid.defaultGrid.SpacingZ = value; } }
+		public static float				UniformSnapDistance		{ get { return Grid.defaultGrid.SpacingX; } set { Grid.defaultGrid.SpacingX = value;
+																										  Grid.defaultGrid.SpacingY = value;
+																										  Grid.defaultGrid.SpacingZ = value; } } 
```

Comment:
we really need to find a solution for our whitespaces ...

Reply:
I'm not a fan of the way this block is laid out much at all honestly, it heavily relies on the fact each thing is a single line but getter/setters shouldn't be assumed to be 1-line